aten/src/ATen/native/TensorShape.cpp

```diff
@@ -3651,6 +3651,17 @@ Tensor view(const Tensor& self,
 }
 
 Tensor alias(const Tensor& self) {
+  switch (self.layout()) {
```

Comment:
Why not use `native_functions.yaml` specific keys to call these ops?

Reply:
Fixed.